components/scream/src/share/io/scream_output_manager.cpp

```diff
@@ -392,6 +392,12 @@ set_params (const ekat::ParameterList& params,
 
     m_output_file_specs.max_snapshots_in_file = m_params.get<int>("Max Snapshots Per File");
     m_casename = m_params.get<std::string>("Casename");
+
+    // Allow user to ask for higher precision for normal model output,
+    // but default to single to save on storage
```

Comment:
Since this line still applies, why does this pass baseline comparison.  Shouldn't the new output all be SP?

Reply:
Good question. I don't know.

Reply:
Perhaps cprnc casts everything to whatever the worst precision is. Or maybe it uses the precision of the 1st file (which happens to be from this PR)

Reply:
AH, indeed! Running cprnc with the files swapped causes the test to fail!